editor/editor_themes.cpp

```diff
@@ -255,6 +255,27 @@ static Ref<ImageTexture> editor_generate_icon(int p_index, float p_scale, float
 	return ImageTexture::create_from_image(img);
 }
 
+float get_gizmo_handle_scale(const String &gizmo_handle_name = "") {
+	const float scale_gizmo_handles_for_touch = EDITOR_GET("interface/touchscreen/scale_gizmo_handles");
+	if (scale_gizmo_handles_for_touch > 1.0f) {
+		// The names of the icons that require additional scaling.
+		HashSet<StringName> gizmo_to_scale;
+
+		gizmo_to_scale.insert("EditorHandle");
+		gizmo_to_scale.insert("EditorHandleAdd");
+		gizmo_to_scale.insert("EditorHandleDisabled");
+		gizmo_to_scale.insert("EditorCurveHandle");
+		gizmo_to_scale.insert("EditorPathSharpHandle");
+		gizmo_to_scale.insert("EditorPathSmoothHandle");
```

Comment:
You are re-creating this set for every call of this method. It should be created only once.
If you want it to be local, make it static and populate if empty.